### PR TITLE
Skipping vaults from download command

### PR DESCRIPTION
**Pull request by kuzentio**
Skipping Vaults for being downloaded when download recursively.

### CODE REVIEW

File: solvebio/cli/data.py [hunk 1]
```diff
@@ -645,6 +645,9 @@ def _download_recursive(
         file_obj.depth = depth
         # MD5 not retrieved by GlobalSearch so
         # separate API call is needed
+        if file_obj.class_name() == "Vault":
+            print("Do not processing Vaults, skipping {}".format(file_obj['vault']))
```
Comment (nishanthmerwin) [inline solvebio/cli/data.py:649]: I don't think we need to print to warn users. We should just silently skip it.

File: solvebio/version.py [hunk 2]
```diff
@@ -3,4 +3,4 @@
 
 # This file should define a variable VERSION which we use as the
 # debugger version number.
-VERSION = '2.21.0'
+VERSION = '2.21.1'
```
Comment (nishanthmerwin) [inline solvebio/version.py:6]: Can you use bumpversion to increment the patch version?

**Comment by nishanthmerwin** [comment]
@kuzentio the code changes look great. Can we add a test case to ensure this is fixed?

I would just add to here: https://github.com/solvebio/solvebio-python/blob/master/solvebio/test/test_shortcuts.py#L522
Add another entry with [fake_vault_create](https://github.com/solvebio/solvebio-python/blob/6d71d448dcb5389071cc34531cf71a7743aa6eae/solvebio/test/client_mocks.py#L239)

File: requirements-dev.txt [hunk 3]
```diff
@@ -17,3 +17,4 @@ pyyaml==5.3.1
 click==7.1.2
 ruamel.yaml==0.16.12
 pytest
+requests
```
[the remark this answers is not in the document]
Comment (nishanthmerwin) [reply requirements-dev.txt:20]: https://github.com/solvebio/solvebio-python/blob/master/setup.py#L28

How did you install the package? Pretty sure it already installs requests (see link above) if you use `pip install -e .`

File: .bumpversion.cfg [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.21.0
+current_version = 2.22.0
```
Comment (davecap) [inline .bumpversion.cfg:2]: we should bump version separately

**Comment by davecap** [comment]
@kuzentio @nishanthmerwin I'm honestly not sure what changed... maybe github changed the way their action runners work because they used to just run on github. Now it is saying we don't have the ability to use github-hosted runners.

**Comment by nishanthmerwin** [comment]
> @kuzentio @nishanthmerwin I'm honestly not sure what changed... maybe github changed the way their action runners work because they used to just run on github. Now it is saying we don't have the ability to use github-hosted runners.

Found the error: https://stackoverflow.com/a/70968478
Looks like we just need to bump up ubuntu. They discontinued it on April 1st!

**Comment by davecap** [comment]
@nishanthmerwin nice!!!

**Comment by nishanthmerwin** [comment]
Note that a _lot_ of things start breaking in ubuntu 22. Pretty much all the python 2 stuff stops working. I don't think github has announced when they'll start taking it down, but expect we have ~2 years left for this to work before we have to stop supporting python 2.

**Review by nishanthmerwin — APPROVED** [review]
Removed the version bump within this PR. Can do this afterwards.

**Pull request merged**